python/TestHarness/testers/Tester.py

```diff
@@ -41,6 +41,8 @@ def validParams():
     params.addParam('petsc_debug',   ['ALL'], "{False,True} -> test only runs when PETSc is configured with --with-debugging={0,1}, otherwise test always runs.")
     params.addParam('curl',          ['ALL'], "A test that runs only if CURL is detected ('ALL', 'TRUE', 'FALSE')")
     params.addParam('tbb',           ['ALL'], "A test that runs only if TBB is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('pthreads',      ['ALL'], "A test that runs only if PTHREADS is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('openmp',        ['ALL'], "A test that runs only if OPENMP is available ('ALL', 'TRUE', 'FALSE')")
```

Comment:
The problem stems from libMesh having multiple threading models "enabled", but only using one. 

If you think of these flags with respect to how libMesh views the models, then this is consistent.
    - `tbb = True` only runs when TBB is enabled in libmesh
    - `pthreads = True` only runs when pthreads is enabled in libmesh
    - 'openmp = True' ...

The flags say nothing of which model is being used, just what is enabled in libMesh. I can come up with something better, but we would have to hard-code the assumed libMesh threading model selection which seems like a bad idea. Also, we use these flags so infrequent I am not sure it is worth the effort.